lib/game.rb

```diff
@@ -0,0 +1,74 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2, :cells
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+    ]
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2
+  end
+
+
+  def won?
+     WIN_COMBINATIONS.detect do |combo|
+      @board.cells[combo[0]] == @board.cells[combo[1]] && @board.cells[combo[1]] == @board.cells[combo[2]] && @board.cells[combo[0]] != " "
+    end
+  end
+
+  def draw?
+    !won? && @board.cells.all?{|i| i == "X" || i == "O"}

```

Comment:
This looks suspiciously familiar...is there another method that you have already defined which you could call here?